### PR TITLE
Feat MGet

### PR DESCRIPTION
**Pull request by daoshenzzg**
Motivation:

We need to support the function of batch ID querying cache entities. Golang's generics can support this function very conveniently.

### CODE REVIEW

File: remote/goredisv8adapter.go [hunk 1]
```diff
@@ -40,6 +43,47 @@ func (r *GoRedisV8Adaptor) Del(ctx context.Context, key string) (val int64, err
...
+		logger.Error("MGet:pipeline.Exec error(%v)", err)
+	}
+
+	for idx, cmd := range cmder {
+		if strCmd, ok := cmd.(*redis.StringCmd); ok {
+			key := keyIdxMap[idx]
+			val, err := strCmd.Result()
+			if err != nil && !errors.Is(err, r.Nil()) {
+				logger.Error("MGet#strCmd(%s) error(%v)", keyIdxMap[idx], err)
+			} else if len(val) > 0 {
```
Comment (ikenchina) [inline remote/goredisv8adapter.go:67]: 万一是空呢，而不是nil

File: remote/goredisv8adapter.go [hunk 2]
```diff
@@ -40,6 +43,47 @@ func (r *GoRedisV8Adaptor) Del(ctx context.Context, key string) (val int64, err
 	return r.client.Del(ctx, key).Result()
 }
 
+func (r *GoRedisV8Adaptor) MGet(ctx context.Context, keys ...string) (map[string]any, error) {
+	pipeline := r.client.Pipeline()
+	keyIdxMap := make(map[int]string, len(keys))
+	ret := make(map[string]any)
```
Comment (ikenchina) [inline remote/goredisv8adapter.go:49]: 对于这种sdk，最好赋值cap

File: cache.go [hunk 3]
```diff
@@ -294,21 +296,150 @@ func (c *jetCache) Delete(ctx context.Context, key string) error {
...
+
+	if len(missIds) > 0 && fn != nil {
+		c.statsHandler.IncrQuery()
+
+		fnValues, err := fn(ctx, missIds)
+		if err != nil {
+			c.statsHandler.IncrQueryFail(err)
+			logger.Error("MGet#fn(%s) error(%v)", util.JoinAny(",", ids), err)
+		} else {
+			placeholderValues := make(map[string]any)
```
Comment (ikenchina) [inline cache.go:316]: 设置下cap？

File: remote/goredisv8adapter.go (hunk 2 above)
Comment (daoshenzzg) [reply remote/goredisv8adapter.go:49]: OK，统一加上，避免扩容。

**Pull request merged**